src/strategies/sd-reverse-delegation/index.ts

```diff
@@ -0,0 +1,138 @@
+import { getDelegations } from '../../utils/delegation';
+import { getScoresDirect } from '../../utils';
+import { multicall } from '../../utils';
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
+import { getAddress } from '@ethersproject/address';
+
+export const author = 'pierremarsotlyon1';
+export const version = '0.1.0';
+export const dependOnOtherAddress = true;
```

Comment:
Hi @pierremarsotlyon1 Right now we are not accepting strategies that depend on other addresses to calculate voting power. these types of strategies are usually not scalable, you see some other way to handle this without depending on other voters

Reply:
https://github.com/snapshot-labs/snapshot-strategies#code

Reply:
Hey @ChaituVR 
I didn't really understand the issue here
I'm just calculating the voting power of each voters to calculate the voting power of another voter
Which is basically like calculate the voting power of each voters like all the other strategies

For example, if we have these voters : [0x1, 0x2, 0x3] and the strategy will want to calculate the voting power of 0x3 (based on strategy parameters), I will calculate the voting power of 0x1 and 0x2 to assign the voting power of 0x3

What's the issue here ?

Reply:
Yes we do have other 15 old strategies that depend on other addresses, but we stopped accepting new ones as they are not scalable and causing lot of trouble, one alternative way is running seperate server to return the voting power of addresses and using the strategy `api-v2-override`

Reply:
But there is something that i don't understand
Currently, with your current default delegation strategy, how you reduce the delegation address voting power if there are some voters who delegated to the delegation address ?

Reply:
> Currently, with your current default delegation strategy, how you reduce the delegation address voting power if there are some voters who delegated to the delegation address?

We calculate voting power for all addresses again (that's the pain problem 🙈 ), and then we will just ignore the voting power of the delegator who voted from the delegate's voting power

Reply:
Is it possible to calculate in a external server the voting power of each user and then compute the delegation address voting power in the strategy ?

the delegation address voting power will still be based on the voter addresses but at least the user voting power will be compute externally
So basically, the strategy will just fetch voting powers with an endpoint and then do plus/minus stuff

Reply:
Your strategy can do anything within limits mentioned here https://github.com/snapshot-labs/snapshot-strategies#code just that depending on other addresses is not scalable for us

Reply:
Ok, we're blocked with what we want to achieve and your requirements.
Is it really not possible to push this strat ?

Otherwise, is it possible to use your delegation strategy (like currently we have) but use a custom delegation contract address by the strategy settings ? 
Or is it possible to add this logic on your side ?

Basically, our "custom" delegation contract will be the same as yours but with new functions